src/main/java/bio/terra/service/resourcemanagement/azure/AzureMonitoringService.java

```diff
@@ -0,0 +1,626 @@
+   * will alert if too many unauthorized access attempts are made on said Storage Account
+   *
+   * @param profileModel The billing profile for the dataset or snapshot being logged
+   * @param storageAccount The Storage Account being monitored
+   * @return A {@link ScheduledAlertRule} object representing the Sentinel rule that was created
+   */
+  public AlertRule getSentinelRuleUnauthorizedAccess(
+      BillingProfileModel profileModel, AzureStorageAccountResource storageAccount) {
+    SecurityInsightsManager client =
+        resourceConfiguration.getSecurityInsightsManagerClient(
+            resourceConfiguration.getCredentials().getHomeTenantId(),
+            profileModel.getSubscriptionId());
+    try {
+      AlertRule alertRule =
+          client
+              .alertRules()
+              .get(
+                  storageAccount.getApplicationResource().getAzureResourceGroupName(),
+                  storageAccount.getName(),
+                  UNAUTHORIZED_ACCESS_ALERT_NAME);
+      logger.debug("Found Sentinel UnauthorizedAccess alert rule");
+      return alertRule;
+    } catch (ManagementException e) {
+      logger.debug("No Sentinel UnauthorizedAccess alert rule found", e);
+      if (Objects.equals(e.getValue().getCode(), RESOURCE_NOT_FOUND_CODE)
+          || Objects.equals(e.getValue().getCode(), NOT_FOUND_CODE)
+          || (Objects.equals(e.getValue().getCode(), BAD_REQUEST_CODE)
+              && e.getMessage().contains("is not onboarded to Microsoft Sentinel"))) {
+        return null;
+      } else {
+        throw e;
+      }
+    }
+  }
+
+  /**
+   * Create the Sentinel alert rule for the Sentinel instance monitoring the given Storage Account
+   * that will alert if too many unauthorized access attempts are made on said Storage Account
+   *
+   * @param profileModel The billing profile for the dataset or snapshot being logged
+   * @param storageAccount The Storage Account being monitored
+   * @return A string representing the id of the Sentinel rule that was created
+   */
+  public String createSentinelRuleUnauthorizedAccess(
+      BillingProfileModel profileModel, AzureStorageAccountResource storageAccount) {
+    SecurityInsightsManager client =
+        resourceConfiguration.getSecurityInsightsManagerClient(
+            resourceConfiguration.getCredentials().getHomeTenantId(),
+            profileModel.getSubscriptionId());
+
+    // Note: this is copied from the rule defined in the Terra landing zone service
+    logger.info(
+        "Creating new Sentinel rule for Sentinel instance monitoring Storage Account {} - UnauthorizedAccess",
+        storageAccount.getStorageAccountId());
+    return client
+        .alertRules()
+        .createOrUpdate(
+            storageAccount.getApplicationResource().getAzureResourceGroupName(),
+            storageAccount.getName(),
+            "UnauthorizedAccess",
```

Comment:
Is this different from `UNAUTHORIZED_ACCESS_ALERT_NAME`?

Reply:
sigh...it is not.  good catch